### DISCUSSION TITLE
What's the next big thing for openvslam?

### DISCUSSION THREAD

**Discussion by junzhang3141** [stella-cv/stella_vslam#98] [Q&A] [answered]
So what's the plan?
I guess we should first make sure the project is actually good under the BSD 2 License? I am also working on supporting omnidirectional camera model but I guess to remove any possible violation of ORB-SLAM2 is the top priority in the next step. 
Does anyone has a plan yet?

**Comment by ymd-stella** [comment, marked as answer]
For now, the tasks listed at https://github.com/OpenVSLAM-Community/openvslam/discussions/98 are all we have.
